### ISSUE TITLE
Feature Request: Custom bonus/penalty on saving throws and skills

### ISSUE THREAD

**Issue by RecursiveStar** [daft-develop/Tetra-cube.github.io#6]
If you're interested in adding things, here's a suggestion: add a field to allow an optional bonus/penalty for saving throws and skills, on top of the bonus from proficiency or expertise.

To illustrate why I'm requesting this, here's part of the stat block for Manshoon from Waterdeep: Dragon Heist.
![f275875e19d127c026eef1291ef954fb](https://user-images.githubusercontent.com/38794241/91360249-13cc8c00-e7c4-11ea-830c-17de9f147f96.png)

It is impossible to replicate Manshoon's saving throw bonuses (which he obtains from his equipment) using the stat block generator. There is only the option to add proficiency.

The same goes for skills (although you can add expertise for skills) - so this option would be nice, if possible.

**Comment by RecursiveStar** [comment]
@daft-develop I'm going to assume that you don't have an issue with feature requests? I have a couple suggestions that I think would be pretty cool to add to stat blocks, if you're okay with that. Your work so far is great, though :)

**Comment by daft-develop** [comment]
Yeah, I don't mind at all, just no guarantee when I'll get around to them.

I'm mostly doing them as a refresher with basic js and web stuff. I don't have any long term plans for forking this project and don't use it very often these days, but limited scope requests with clear examples are easy to implement so they're fun to do once in a while.

I was hoping the PR would eventually get pulled over so it's not a duplicate repo for the other subprojects Tetra has. If I start tackling different features, I'll probably create separate branches for each to make them easy to PR back, but eventually it could become a bunch of work to rebase periodically. Once rebasing becomes difficult I'm sure my enthusiasm will drop off.

Feel free to create a bunch of issues/requests and I can take a look. I'll probably end up merging all the features back onto my master branch as a more functional consolidation and then figure out how to host just the updated stat block page via GitHub.
